README.md

```diff
@@ -2,6 +2,9 @@
 
 An effort to create a standard to promote lattice information exchange for particle accelerators.
 
+[Access the online documents:](https://campa-lattice.readthedocs.io)
+[![Documentation Status](https://readthedocs.org/projects/campa-lattice/badge/?version=latest)](https://campa-lattice.readthedocs.io)
+
```

Comment:
Still thinking that maybe we don't need the link explanation here and we can just put this badge (and possibly others in the future) at the top of the README file before the rest of the content (like we do in WarpX). I think it should be fine, the link to the online docs is anyways in the "about" description in the main repository page.

Otherwise, we could maybe shorten the text to something more concise like "Online documentation:" or "Online documents:".

Reply:
Agreed, let's do that once we add another badge?